hooks/026-configure-plymouth.chroot

```diff
@@ -0,0 +1,109 @@
+#! /bin/sh -ex
+
+# Note that we have overrides for plymouth services in:
+# static/usr/lib/systemd/system/plymouth-start.service.d/core-override.conf
+# static/usr/lib/systemd/system/plymouth-quit.service.d/core-override.conf
+# static/usr/lib/systemd/system/plymouth-halt.service.d/core-override.conf
+# static/usr/lib/systemd/system/plymouth-reboot.service.d/core-override.conf
+# static/usr/lib/systemd/system/plymouth-kexec.service.d/core-override.conf
+# static/usr/lib/systemd/system/plymouth-poweroff.service.d/core-override.conf
+
+cat << 'EOF' > /lib/systemd/system/splash-client.service
```

Comment:
Those files are not dynamic. Could we move them into `static/`?

Reply:
Maybe we can keep the configuration files here. But the systemd units would be better in `static/`.

Reply:
Done now